src/components/Typography/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "composite": true
   },
   "include": ["src/**/*"],
-  "exclude": ["dist", "src/*.stories.tsx"],
+  "exclude": ["dist", "src/*.stories.tsx", "src/**/*.stories.tsx"],
```

Comment:
`src/**/*.stories.tsx` should be enough. I believe you can remove the exclude in the middle.